node/pkg/common/chainlock.go

```diff
@@ -35,6 +34,10 @@ type MessagePublication struct {
 	Unreliable bool
 }
 
+func (msg *MessagePublication) TxIDString() string {
+	return "0x" + hex.EncodeToString(msg.TxID)
+}
+
 func (msg *MessagePublication) MessageID() []byte {
 	return []byte(msg.MessageIDString())
```

Comment:
Could this return a copy of `msg.TxID` instead? I'm unclear on why this would go through an encoding process and back.

Reply:
This returns the message ID (chain/emitter/sequence) not the transaction ID. (A) I don't want to change that behavior and (B) there could be multiple wormhole messages in a single transaction so TxID may not uniquely identify a message.